subprojects/docs/src/snippets/tutorial/configureObject/kotlin/build.gradle.kts

```diff
@@ -1,12 +1,12 @@
-import java.text.FieldPosition
+class UserInfo(var name: String? = null, var email: String? = null)
 
 tasks.register("configure") {
+    val user = configure(UserInfo(), closureOf<UserInfo> {
```

Comment:
I don't think we should promote `configure` in Kotlin when Kotlin's own `apply` scope function is a better alternative since it doesn't require a Groovy Closure. How about using `apply` here with a comment saying that in most scenarios `configure` has  no benefit over Kotlin's `apply`?

Reply:
Yeah, that is how it was - Groovy also has `tap`, I was thorn between moving Groovy to `tap` or moving Kotlin to `configure` (for the case the object implements `Configurable`), either way would make the two examples a bit more consistent.

I am fine with reverting.

Reply:
However, it feels silly - if we just use `apply`, there is nothing Gradle related in the Kotlin example. Removing the kotlin snippet altogether may make sense here.

Reply:
I think the point of the Kotlin example would be to show there's no API gap because Kotlin's `apply` is a better alternative.

Reply:
Right, minus special support for `Configurable`. Also, note Groovy doesn't *need* it either (Groovy's `tap` provides the same value as `apply`). Anyway, the example is using `apply` now.